scripts/build-rhcc-image.sh

```diff
@@ -83,24 +110,47 @@ else
     INTERNAL_IMAGE_NAME=cb-rhcc/sync-gateway
 fi
 
+BUILD_ARGS="--build-arg PROD_VERSION=${VERSION} --build-arg STAGING=${STAGING}"
+
 # Build and push images
 for registry in ghcr.io build-docker.couchbase.com; do
     IMAGE=${registry}/${INTERNAL_IMAGE_NAME}:${VERSION}-${BLD_NUM}
 
     ${SCRIPT_DIR}/update-base.sh ${DOCKERFILE}
-    echo @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
-    echo Building ${IMAGE}
-    echo @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
-    docker build ${CACHE_ARG} \
-      --build-arg PROD_VERSION=${VERSION} \
-      --build-arg STAGING=${STAGING} \
-      -f ${DOCKERFILE} -t ${IMAGE} .
-
-    if [ "x${DRYRUN}" != "xyes" ]; then
+    if [ "${DRYRUN}" = "yes" ]; then
+        IMAGE=${registry}/${INTERNAL_IMAGE_NAME}:${VERSION}-${BLD_NUM}
+        # For dry run where buildx is present and we're doing a multiarch
+        # build, we build each architecture's image individually so we
+        # can load them into the local image store for testing
+        if multiarch; then
+            for arch in amd64 arm64; do
+            echo @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+            echo Building ${IMAGE}-${arch}
+            echo @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+                docker buildx build --platform linux/${arch} --load \
```

Comment:
You can (and probably should) just do "docker buildx build --platform linux/arm64,linux/amd64" here, rather than looping.

Reply:
This is for dry-runs, the loop's required so the resulting images can be `--load`ed.

Reply:
Ah, yes, OK. Got it.